### ISSUE TITLE
Impetus drainer and diffuser animations stutter when moving far away from them 

### ISSUE THREAD

**Issue by Turkey9002** [TheCodex6824/ThaumicAugmentation#193]
https://user-images.githubusercontent.com/86805568/124369693-55279680-dc34-11eb-8337-81b9c8d0657e.mp4

Any chunks you leave off and come back to with existing drainers or diffusers start to stutter on their animations while any newly placed ones will animate normally. One interesting thing I've noticed is that leaving and re-entering the world fixes it while reloading the chunks does not. I've also tested this without any other mods to make sure those are not causing the problem.

**Comment by TheCodex6824** [comment]
Yeah, the animations on some blocks stutter a bit in general when starting / stopping. At some point I have to go back and try to fix that - I keep putting it off because if I can't fix them in the current system then I have to completely redo them, which would not be very fun.
